rest-service/manager_rest/server.py

```diff
@@ -151,6 +170,55 @@ def reset_state(configuration=None):
     app = setup_app()
 
 
+def validate_security_config(cfy_config):
+    # TODO raise better exceptions
+    if not hasattr(cfy_config, 'securest_secret_key') or \
+            cfy_config.securest_secret_key is None:
+        raise Exception('securest_secret_key not set')
+    if not hasattr(cfy_config, 'securest_authentication_methods') or \
+            cfy_config.securest_authentication_methods is None:
+        raise Exception('securest_authentication_methods not set')
+    if not hasattr(cfy_config, 'securest_userstore_driver') or \
+            cfy_config.securest_userstore_driver is None:
+        raise Exception('securest_userstore_driver not set')
+    if not hasattr(cfy_config, 'securest_userstore_identifier_attribute') or \
+            cfy_config.securest_userstore_identifier_attribute is None:
+        raise Exception('securest_userstore_identifier_attribute not set')
+
+
+def init_secure_app(app):
+    cfy_config = config.instance()
+    validate_security_config(cfy_config)
+
+    app.config[SECUREST_SECRET_KEY] = cfy_config.securest_secret_key
```

Comment:
revisit userstore+properties to configure it (same goes for authentication methods + their configuration)